README.md

```diff
@@ -1 +1,2 @@
-# Hello World!!! - Krista
\ No newline at end of file
+# Hello World!!! - Krista
+# Curious how merge conflicts work
```

Comment:
Github has a nice online editor to resolve simple merge conflicts

Reply:
I saw that, cool!